README.md

```diff
@@ -332,8 +332,8 @@ source that outputs ETI-NI. The respective binary is then called with
 the necessary parameters, including the frequency and an optional gain
 value.
 
-You therefore just have to specify the path to the `dab2eti` binary and
-the desired channel.
+You therefore just have to specify the absolute path to the `dab2eti` 
```

Comment:
There is no need for an absolute path here, as the DAB live source executable may be within `$PATH` and hence accessible from everywhere.

Reply:
Thanks, it even works without the path, if it is in $PATH, BTW.

Reply:
Well, actually this is exactly what I wrote...